db/migrate/20220705142852_add_user_id_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :user_id, :integer
```

Comment:
Please add `author_id` instead of `user_id` to posts table as it can be seen in this [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).